src/sortStudents.ts

```diff
@@ -1,16 +1,66 @@
+/* eslint-disable default-case */
 
 export interface Student {
-  // describe Student interface
+  name: string,
+  surname: string,
+  age: number,
+  married: boolean,
+  grades: number[],
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name,
+  Surname,
+  Age,
+  Married,
+  AverageGrade,
 }
 
 // create SortOrder type
-export type SortOrder;
+export type SortOrder = {
+  asc : string,
+  desc: string,
+};
```

Comment:
I suggest rewrite in this way:
```suggestion
export type SortOrder  = 'asc' | 'desc';
```